### PR TITLE
updates readme with slack integration test

### PR DESCRIPTION
**Pull request by WilliamParty**
This is just to test slack

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,4 +6,4 @@ No you don't, but I (the one who forked) must remember to setup an upstream to t
 
 Now I'll do a pull request, and while waiting for approval do some other work.
 
-
+I'm gonna try slack integration here.
```
Comment (WilliamParty) [inline README.md]: It seems to work

Comment (WilliamParty) [inline README.md]: And it really does

**Comment by mpmobile** [comment]
Pull request accepted, please proceed with merge

**Comment by mpmobile** [comment]
Will slack show this message?

File: README.md (hunk 1 above)
Comment (WilliamParty) [inline README.md]: A line note, will slack show it?

Comment (WilliamParty) [inline README.md]: No it doesn't

Comment (WilliamParty) [inline README.md]: As a matter of fact, it does :dancer:

**Pull request merged**